src/preact/slot.js

```diff
@@ -20,12 +20,15 @@ const EMPTY = {};
 /** @const {WeakMap<Element, {oldDefauls: (!Object|undefined), component: Component}>} */
 const cache = new WeakMap();
 
+/** @typedef {import('preact').VNode} VNode */
+/** @typedef {import('preact').FunctionComponent} FunctionComponent */
+
 /**
- * @param {!Element} element
+ * @param {Element} element
  * @param {string} name
- * @param {!Object|undefined} defaultProps
+ * @param {Object|undefined} defaultProps
  * @param {boolean|undefined} as
- * @return {!PreactDef.VNode|!PreactDef.FunctionalComponent}
+ * @return {VNode | FunctionComponent}
```

Comment:
For consistency's sake only
```suggestion
 * @return {VNode|FunctionComponent}
```

Reply:
Done